website/src/utils/isStringArray.ts

```diff
@@ -0,0 +1,8 @@
+export function isStringArray(obj: unknown): obj is string[] {
+  if (typeof obj === 'object' && Array.isArray(obj)) {
+    return obj.reduce(
+      (acc: boolean, el: unknown) => !(acc && typeof el != 'string'),
```

Comment:
This might be easier to read as `return !obj.find(el =>typeof el !== 'string')`

Reply:
That is true. It looks a lot cleaner! I'll make edits along with Zod before submitting another pull request. I'll reject this one.